cluster-autoscaler/main.go

```diff
@@ -170,7 +170,7 @@ var (
 			"Setting it to false employs a more lenient filtering approach that does not try to pack the pods on the nodes."+
 			"Pods with nominatedNodeName set are always filtered out.")
 
-	ignoreTaintsFlag         = multiStringFlag("ignore-taint", "Specifies a taint to ignore in node templates when considering to scale a node group")
+	ignoreTaintsFlag         = multiStringFlag("ignore-taint", "Specifies a taint to ignore when considering to scale a node group and mark the existing nodes as NotReady when they're still tainted")
```

Comment:
Marking as NotReady is an implementation detail (hack really) that is only used for internal bookkeeping. I'd leave it out of a flag value as it's not relevant to what the flag does and the way it's phrased suggests CA is updating actual Node objects in cluster  and not just in-memory state in CA.

Reply:
Completely make sense 👍